.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Java CI with Maven
+
+on:
+  push:
+    tags: [ master, 2.3.x ]
+
+jobs:
+  build:
+    name: "Release"
+    strategy:
+      matrix:
+        version: [ 8.0.272.hs-adpt ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-maven-
+      - name: Download ${{ matrix.version }}
+        uses: sdkman/sdkman-action@master
+        id: sdkman
+        with:
+          candidate: java
+          version: ${{ matrix.version }}
+      - name: Set up ${{ matrix.version }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+          jdkFile: ${{ steps.sdkman.outputs.file }}
+          server-id: bintray-awspring
+          server-username: BINTRAY_USERNAME
+          server-password: BINTRAY_PASSWORD
+      - name: Build with Maven
+        run: ./mvnw -V verify
+      - name: Build with Maven
+        run: ./mvnw -V deploy
+        env:
+          BINTRAY_USERNAME: awspring-releaser
```

Comment:
tbd /cc @maciejwalkowiak

Reply:
The org name is awspring https://bintray.com/awspring

Reply:
this is the username to perform de deploy, can be you or me or a new user we want to create.

Reply:
I guess it's the best to create a new user. Can you take care of it?

Reply:
sure!